prisma/schema.prisma

```diff
@@ -60,11 +60,13 @@ model Session {
 }
 
 model ShowGuest {
-  id      String @id @default(uuid())
-  showId  String
-  guestId String
-  Show    Show   @relation(fields: [showId], references: [id])
-  Guest   Guest  @relation(fields: [guestId], references: [id])
+  id           String      @id @default(uuid())
+  showId       String
+  guestId      String
+  Show         Show        @relation(fields: [showId], references: [id])
+  Guest        Guest       @relation(fields: [guestId], references: [id])
+  Transcript   Transcript? @relation(fields: [transcriptId], references: [id])
```

Comment:
Since this is just a table to relate a specific show with a specific guest, is having Transcript referenced here essential?

Reply:
Ah I initially had a relation between the guest and the utterance. I scrapped it for now so I'll remove